tests/frontend/test_ops_decompositions.py

```diff
@@ -593,50 +525,46 @@ def test_decomposition(self, hbar, tol):
         assert np.allclose(cov, cov_res, atol=tol, rtol=0)
 
     def test_thermal_decomposition(self, hbar, tol):
-        """Test that an thermal state decomposition provides correct covariance matrix"""
+        """Test that a thermal covariance matrix decomposes into Thermal preparations."""
         n = 3
-        prog = sf.Program(n, hbar=hbar)
+        prog = sf.Program(n)
         nbar = np.array([0.453, 0.23, 0.543])
         cov = np.diag(np.tile(2 * nbar + 1, 2)) * hbar / 2
 
-        with eng:
-            G = ops.Gaussian(cov)
-            cmds = G.decompose(q)
+        G = ops.Gaussian(cov)
+        cmds = G.decompose(prog.register)
 
         assert len(cmds) == n
 
         # calculating the resulting decomposed symplectic
         for i, cmd in enumerate(cmds):
-            # all operations should be Thermal states
             assert isinstance(cmd.op, ops.Thermal)
             assert np.allclose(cmd.op.p[0].x, nbar[i], atol=tol, rtol=0)
 
     def test_squeezed_decomposition(self, hbar, tol):
-        """Test that an squeeze state decomposition provides correct the covariance matrix"""
+        """Test that an axially squeezed covariance matrix decomposes into Squeezed preparations."""
         n = 3
-        prog = sf.Program(n, hbar=hbar)
+        prog = sf.Program(n)
 
         sq_r = np.array([0.453, 0.23, 0.543])
         S = np.diag(np.exp(np.concatenate([-sq_r, sq_r])))
-        cov = S @ S.T
+        cov = S @ S.T * (hbar / 2)
```

Comment:
Interesting 🤔 
Why did hbar need to be inserted here, while it was largely dropped everywhere else?

Reply:
It's because unlike symplectic transformations, covariance matrices do depend on the value of the vacuum fluctuations (=hbar)

Reply:
ok, in that case, then why wasn't the hbar used before?

Reply:
Probably because a particular value of hbar was assumed (likely hbar = 2)

Reply:
Weird that this never broke when we changed hbar=1.7

Reply:
Yeah that's the reason.